Cython/Compiler/ExprNodes.py

```diff
@@ -9850,10 +9857,18 @@ class GeneratorExpressionNode(LambdaNode):
     #
     # loop      ForStatNode   the for-loop, containing a YieldExprNode
     # def_node  DefNode       the underlying generator 'def' node
+    # call_parameters [ExprNode]   (Internal) parameters passed to the DefNode call
```

Comment:
I wonder if there can be more than one argument. In generator expressions, only the outermost iterable is special.
Are you anticipating optimisations for `zip()` or something like that?
If we can stick to one argument, that seems simpler (and gives us a standard `METH_O` signature, I think, right?).

Reply:
> I wonder if there can be more than one argument. 

Yes - quite frequently. Say:

```
return (x for x in range(func1(), func2()))
```

You should probably end up with arguments for `result_of_func1` and `result_of_func2` due to the optimizations. Obviously it'd be fairly easy to block the optimization and up up with one argument, but I got the impression that this wasn't popular!

>  If we can stick to one argument, that seems simpler (and gives us a standard METH_O signature, I think, right?).

It's called as a C function rather than a Python function so I don't think keeping the signature is hugely important.